web/static/js/components/layout/HeaderContainer.jsx

```diff
@@ -48,22 +47,39 @@ HeaderContainer.propTypes = {
   tabs: PropTypes.node,
   logout: PropTypes.func.isRequired,
   user: PropTypes.string.isRequired,
-  surveyFolder: PropTypes.object
+  surveyFolder: PropTypes.object,
+  panelSurvey: PropTypes.object
 }
 
 const mapStateToProps = (state, ownProps) => {
+  const { params } = ownProps
   const folders = state.folder && state.folder.folders
   const survey = state.survey && state.survey.data
-  const surveyFolder = survey && folders && folders[survey.folderId]
+  const panelSurveyId = params.panelSurveyId && parseInt(params.panelSurveyId)
+
+  let panelSurvey = null
+  if (state.panelSurvey && state.panelSurvey.id == panelSurveyId) {
+    panelSurvey = state.panelSurvey
+  }
```

Comment:
**I** prefer `const`s to `let`s when possible

Reply:
Yeah! I prefer `const` too!! I'll try it harder! 😉